test/test_prototype_transforms.py

```diff
@@ -349,15 +349,15 @@ def test__transform_bounding_box_clamping(self, mocker):
             ),
         )
 
-        bounding_box = make_bounding_box(
+        bounding_boxes = make_bounding_box(
```

Comment:
This call is a bit awkward now.

I didn't want to change `make_bouding_box` into `make_bounding_boxes` because `make_bouding_boxes` already exists.

I'm hoping we'll be able to get rid of the current `make_bouding_boxes` util soon anyway? CC @pmeier